### PR TITLE
Add completion for fish shell

### PR DESCRIPTION
**Pull request by marians**
This is supposed to add fish support for `gsctl completion`.

### CODE REVIEW

File: commands/completion.go [hunk 1]
```diff
@@ -14,23 +14,39 @@ import (
 var (
 	// CompletionCommand is the command to create things
 	CompletionCommand = &cobra.Command{
-		Use:                   "completion <bash|zsh> [--stdout]",
+		Use:                   "completion <bash|fish|zsh> [--stdout]",
 		Args:                  cobra.ExactArgs(1),
 		DisableFlagsInUseLine: true,
-		Short:                 "Create completion file for bash, and (experimentally) for zsh",
+		Short:                 "Create completion file for bash, fish, and for zsh",
```
Comment (axbarsan) [inline commands/completion.go:20]:
```suggestion
		Short:                 "Create completion file for bash, fish, or zsh",
```

File: commands/completion.go [hunk 2]
```diff
@@ -14,23 +14,39 @@ import (
...
+   source <(gsctl completion bash --stdout)
+
+2. Start a new terminal session
+
+Fish
+----
+
+Run
+
+  gsctl completion fish --stdout > ~/.config/fish/completions/gsctl.fish
```
Comment (axbarsan) [inline commands/completion.go:43]: Should this have the same indentation as the example for `Bash` completion?

**Review by kubasobon — APPROVED** [review]
Works, LGTM

**Comment by marians** [comment]
Oops, I was a little too light on the trigger. I meant to act on @axbarsan 's review comments before merging... will follow up.

**Pull request merged**